### ISSUE TITLE
Motion JPEG ffmpeg command colors reverted

### ISSUE THREAD

**Issue by tanmanh0707** [moononournation/RGB565_video#21]
Hi,

Thank you for your great work!
I used your motion Mjpeg ffmpeg command to a video on youtube. But when I played it back, the colors get reverted. This is my ffmpeg version, could you please help me this?

ffmpeg version 6.0-essentials_build-www.gyan.dev Copyright (c) 2000-2023 the FFmpeg developers

Thank you,
Manh

**Comment by moononournation** [comment]
thinking ffmpeg inverted the color is interesting, but in most case it just a display setting. E.g. IPS parameters.

**Comment by tanmanh0707** [comment]
> thinking ffmpeg inverted the color is interesting, but in most case it just a display setting. E.g. IPS parameters.

Oh sorry for misunderstanding. Let me describe it more details.
I downloaded a video from youtube in *.mp4 format, then used ffmpeg command to convert it to 240x240 mjpeg format - video.mjpeg
Then I ran "ffplay.exe video.mjpeg" and it worked properly with correct colors. After that, I copied "video.mjpeg" to sd card and played it back using the esp32 sketch, the video colors got inverted in my display. Are you encounter the same issue?

This is Youtube video that I am using: https://www.youtube.com/watch?v=O9KZgV-_LUo
